### PR TITLE
docs(common): Merge two ngOptimizedImage docs into one

### PR DESCRIPTION
**Pull request by atcastle**
This PR merges the two documentation pages for the new NgOptimizedImage directive. The custom loader setup documentation is now its own section in the main docs, as opposed to being a separate page. CC: @AndrewKushnir @pkozlowski-opensource

### CODE REVIEW

File: aio/content/guide/image-directive.md [hunk 1]
```diff
@@ -188,10 +188,72 @@ The `sizes` attribute in the above example says "I expect this image to be 100 p
 
 For additional information about the `sizes` attribute, see [web.dev](https://web.dev/learn/design/responsive-images/#sizes) or [mdn](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes).
 
+## Configuring an image loader for `NgOptimizedImage`
+
+This section explain how to set up an image loader for use with the `NgOptimizedImage`.
```
Comment (AndrewKushnir) [inline aio/content/guide/image-directive.md:193]: I think we can drop this sentence.

**Review by AndrewKushnir — APPROVED** [review]
👍

File: aio/content/guide/image-directive.md (hunk 1 above)
Comment (atcastle) [reply aio/content/guide/image-directive.md:193]: Sure, done

**Review by AndrewKushnir — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 6438f69f285fc1af0b01f7e76861a952b25569a0.

**Pull request closed without merging**